ansible_collections/arista/avd/roles/eos_designs_to_containerlab/tasks/deploy.yml

```diff
@@ -0,0 +1,237 @@
+---
+#tasks file for containerlab hosts
+
+- ansible.builtin.setup:
```

Comment:
I believe this will be quite slow compared to `!all` since it will still gather everything, but just filter on output.

Reply:
changed to `!all`